metricflow-semantics/metricflow_semantics/test_helpers/semantic_manifest_yamls/simple_manifest/metrics.yaml

```diff
@@ -629,6 +632,16 @@ metric:
       fill_nulls_with: 0
   time_granularity: week
 ---
+metric:
```

Comment:
Can we add a metric-level filter here too to see what happens in a query with both?